Fabric.Authorization.AccessControl/karma.conf.js

```diff
@@ -27,7 +28,7 @@ module.exports = function (config) {
     colors: true,
     logLevel: config.LOG_INFO,
     autoWatch: true,
-    browsers: ['Chrome'],
+    browsers: [ 'Chrome', 'PhantomJS' ],
```

Comment:
PhantomJS has been deprecated in favor of HeadlessChrome.  Is there a particular reason you want to use PhantomJS?

Reply:
No particular reason, and I would be happy to avoid using it. When I was searching around for running karma tests in VSTS builds everything I saw referenced PhantomJS. I will look into HeadlessChrome

Reply:
I believe we have gotten it working in VSTS Hosted agents for Atlas.  Let me check out what we're doing and I we can talk.